db/seeds.rb

```diff
@@ -1,7 +1,41 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+Listing.destroy_all
+User.destroy_all
+
+user = User.create(email: 'user@mail.com', password: 'password')
+subscriber = User.create(email: 'subscriber@mail.com', password: 'password')
+
+free_listing_1 = Listing.create(
+  category: 'Parking spot',
+  lead: 'Takes less than 1 minute to get the metro',
+  scene: 'outdoor',
+  description: 'Curabitur aliquet quam id dui posuere blandit.' * 3,
+  address: 'Björcksgatan 37 B416 52 Göteborg'
```

Comment:
Small thing but it seems the street number and zip code are not separated. Should be "Björcksgatan 37B, 416 52 Göteborg"